torch/quantization/fx/graph_module.py

```diff
@@ -47,8 +44,33 @@ def __deepcopy__(self, memo):
         fake_mod.__dict__ = copy.deepcopy(self.__dict__)
         return ObservedStandaloneGraphModule(fake_mod, self.graph)
 
-def mark_observed_standalone_module(module: GraphModule) -> GraphModule:
-    return ObservedStandaloneGraphModule(module, module.graph)
-
 def is_observed_standalone_module(module: Any) -> bool:
     return isinstance(module, ObservedStandaloneGraphModule)
+
+
+def _save_packed_weight(self, destination, prefix, keep_vars):
+    for attr_name in dir(self):
+        if "_packed_weight" in attr_name and \
+           isinstance(getattr(self, attr_name), torch._C.ScriptObject):  # type: ignore
+            packed_weight = getattr(self, attr_name)
+            destination[prefix + attr_name] = packed_weight
+
+class QuantizedGraphModule(GraphModule):
```

Comment:
would it be possible to add a short docblock on why this class existis, so a code reader could understand it without opening this PR to get the context?

Reply:
Sure